Scripts/script-SendEmailOnSLABreach.yml

```diff
@@ -0,0 +1,68 @@
+commonfields:
+  id: SendEmailOnSLABreach
+  version: -1
+name: SendEmailOnSLABreach
+script: |-
+  """
+  This script is used to send an email about a breached SLA. The script, by default, sends the email to the assignee of the
+  incident. The email is sent using the send-email command, using all enabled integrations that support the command.
+  """
+
+  def get_owner_email():
+      owner_username = demisto.incidents()[0].get("owner")
+      if owner_username:
+          try:
+              owner_info = demisto.executeCommand('getUserByUsername', {"username":owner_username})[0]
+              owner_email = owner_info.get("EntryContext").get("UserByUsername").get("email")
```

Comment:
I think you can use `demisto.get("EC.UBU.Email")`

Reply:
Also, can this be a list?

Reply:
What is that command you mentioned? 
Regarding email being a list - I really don't know. Isn't there one email configured per user?

Reply:
Using a single "get" won't work as there are multiple nested dictionaries and you have to use "get" for each one